README.rst

```diff
@@ -1,5 +1,5 @@
 ===============
-pyxform v1.0.3-oc
+pyxform v1.0.3+oc
```

Comment:
why is "-" not valid?

Reply:
In addition to that (1.0.3-oc) version is rejected by pip install, it probably not complies to
- https://peps.python.org/pep-0440/#public-version-identifiers
- https://peps.python.org/pep-0440/#local-version-identifiers

We can see the summary of possible version number here:
- https://peps.python.org/pep-0440/#examples-of-compliant-version-schemes

Thanks.

Reply:
got it, thanks. In that case, I think it would be better to go with something like v1.0.3.oc instead of v1.0.3+oc.

Reply:
Unfortunately, 1.0.3.oc is not accepted.
![image](https://github.com/OpenClinica/pyxform/assets/60411/1dde85f0-a38a-48c3-b286-b7b926b580df)

Reply:
Public version should complyto: `[N!]N(.N)*[{a|b|rc}N][.postN][.devN]`
Local version should comply to: `<public version identifier>[+<local version label>]`

Should we go with 1.0.3+oc?

Thanks.

Reply:
@gushil yes, please. Thanks for trying the other option.

Reply:
@svadla-oc

Done.